### ISSUE TITLE
[Question] the build failed in the main branch

### ISSUE THREAD

**Issue by stevez** [microsoft/playwright#13373]
branch: main, head( commit c68f57d2d0cb09f3c5e11f52b13635d81af9f3e), when I run 'npm run build', I get the following error:
Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@babel/plugin-proposal-export-namespace-from' imported from ../playwright/babel-virtual-resolve-base.js
    at __node_internal_ (../playwright/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:2601:9)
    at __node_internal_ (../playwright/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:2559:11)
    at new NodeError (../playwright/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:2552:5)
    at packageResolve (../playwright/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:3208:9)
    at moduleResolve (../playwright/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:3242:18)
    at defaultResolve (../playwright/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:3281:13)
    at ../playwright/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:3304:14
    at Generator.next (<anonymous>)
    at asyncGeneratorStep ../playwright/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:63:103)

Also I got following warning:
▲ [WARNING] The CommonJS "module" variable is treated as a global variable in an ECMAScript module and may not work as expected

    packages/playwright-core/src/server/injected/injectedScript.ts:1260:0:
      1260 │ module.exports = InjectedScript;
           ╵ ~~~~~~

  This file is considered to be an ECMAScript module because of the "export" keyword here:

    packages/playwright-core/src/server/injected/injectedScript.ts:75:0:
      75 │ export class InjectedScript {
         ╵ ~~~~~~

**Comment by mxschmitt** [comment]
Try the following: "npm ci && npm run build && npx playwright install && npm run ctest" and make sure you use Node.js 12+ and NPM 8. This works on our bots and should work for you as well!

**Comment by stevez** [comment]
@mxschmitt , thanks for you reply.  I checked my machine: it is node 14.16.1 and npm 6.14.12, after I switched to node v16.14.2 (npm v8.5.0), then I can build successfully. 

I would suggest you mentioned the node version requirement in the github page or contribution guide, or do the version check in the package.json.
and to make it easier, just specify the node version, since usually I use nvm, and npm always follows the node version I choose.
